src/Hedgehog/Range.fs

```diff
@@ -122,7 +122,7 @@ module Range =
             fromBigInt (z + diff)
 
         /// Scale an integral exponentially with the size parameter.
-        let inline scaleExponential (sz0 : Size) (z0 : 'a) (n0 : 'a) : 'a =
+        let inline scaleExponential (lo : 'a) (hi : 'a) (sz0 : Size) (z0 : 'a) (n0 : 'a) : 'a =
```

Comment:
@jacobstanley, I don't like this change, but other alternatives I've tried make the F# compiler infer `'a` as `bigint` and breaks everything. Since `scaleExponential` is `internal` I'll go ahead and merge this.

Right now exponential 64-bit un/signed integers can cause arithmetic overflows, and break upstream packages, so I'll go ahead and proceed with this fix.